packages/web-components/stencil.config.ts

```diff
@@ -35,7 +35,7 @@ export const config: Config = {
   },
   outputTargets: [
     reactOutputTarget({
-      componentCorePackage: '@department-of-veterans-affairs/web-components/dist/types',
+      componentCorePackage: '../dist/types',
```

Comment:
This should probably be reverted. 

Is there anything else that we were throwing against the wall when trying to get this to work that can or should be reverted too?